AnswerKingAPI/tests/Categories/test_view_categories.py

```diff
@@ -0,0 +1,24 @@
+from unittest.mock import patch
+from Categories.view_categories import lambda_handler
+
+@patch('Categories.view_categories.table')
+def test_lambda_handler_success(mock_table):
```

Comment:
The name of your test should consist of three parts: Name of the method being tested. Scenario under which the method is being tested. Expected behavior when the scenario is invoked.